src/store/main/sagas/mainCheckSaga.ts

```diff
@@ -23,6 +25,28 @@ export function* mainCheckSaga(payload: { init:boolean }) {
         },
       },
     },
+    'custom-coinbase': {
```

Comment:
Why Custom-coinbase? Web3Modal support walletlink

Reply:
I have not tested it but can you confirm is it support mobile app?

Reply:
In web3 modal we don't have coinbase support, can u give some examples where u saw it 🙏

Reply:
> In web3 modal we don't have coinbase support, can u give some examples where u saw it 🙏

https://github.com/Web3Modal/web3modal/releases coinbase support

Reply:
Woah awesome, removed the custome code @msoni89, thanks 🙏🏻